internal/ethapi/transaction_args.go

```diff
@@ -154,7 +154,7 @@ func (args *TransactionArgs) setFeeDefaults(ctx context.Context, b Backend) erro
 	}
 	// Now attempt to fill in default value depending on whether London is active or not.
 	head := b.CurrentHeader()
-	if b.ChainConfig().IsLondon(head.Number) {
+	if b.ChainConfig().IsLondon(head.Number) && !args.IsConfidential {
```

Comment:
Do not use London pricing rules for confidential compute requests.

Reply:
Ditto 🙃